plugins/backstage-plugin-coder/src/components/CoderAuthForm/CoderAuthForm.tsx

```diff
@@ -69,17 +47,15 @@ export const CoderAuthWrapper = ({ children, type }: WrapperProps) => {
 
           case 'authenticated':
           case 'distrustedWithGracePeriod': {
-            throw new Error(
-              'Tried to process authenticated user after main content should already be shown',
-            );
+            return <CoderAuthSuccessStatus />;
```

Comment:
I shout this out in one of the comments, but this change was made so that if anything ever goes slightly awry with some of the auth checks, there's no way that we'll throw an error directly in the UI

If all goes well, the user will never see this component, but always good to have some double bookkeeping

Reply:
Good call, makes a lot of sense